code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -234,43 +238,61 @@
 	. = ..()
 	if(!. || ai_inactive) return
 
-	if(prob(5))
+	if(prob(5) && (stance == STANCE_IDLE))
 		lay_down()
 
 //Pet 4 friendly
 
 /mob/living/simple_animal/otie/attack_hand(mob/living/carbon/human/M as mob)
-	..()
-	if(M.a_intent == I_GRAB)
-		if (M == src)
-			return
-		if (!(status_flags & CANPUSH))
-			return
-		if(!incapacitated(INCAPACITATION_ALL) && (stance != STANCE_IDLE) && prob(grab_resist))
-			M.visible_message("<span class='warning'>[M] tries to grab [src] but fails!</span>")
-			return
-		var/obj/item/weapon/grab/G = new /obj/item/weapon/grab(M, src)
-		M.put_in_active_hand(G)
-		G.synch()
-		G.affecting = src
-		LAssailant = M
-		M.visible_message("<span class='warning'>[M] has grabbed [src] passively!</span>")
-		M.do_attack_animation(src)
-		ai_log("attack_hand() I was grabbed by: [M]",2)
-		pixel_x = old_x
-		react_to_attack(M)
-	if(M.a_intent == I_HELP)
-		if (health > 0)
-			LoseTarget()
-			handle_stance(STANCE_IDLE)
-			if(prob(tame_chance))
-				friend = M
-				if(tamed != 1)
-					tamed = 1
-					faction = M.faction
-			sleep(1 SECOND)
-			return
-			..()
+
+	switch(M.a_intent)
+		if(I_HELP)
+			if (health > 0)
+				M.visible_message("<span class='notice'>[M] [response_help] \the [src].</span>")
+				LoseTarget()
+				handle_stance(STANCE_IDLE)
+				if(prob(tame_chance))
+					friend = M
+					if(tamed != 1)
+						tamed = 1
+						faction = M.faction
+				sleep(1 SECOND)
+
+		if(I_DISARM)
+			M.visible_message("<span class='notice'>[M] [response_disarm] \the [src].</span>")
+			M.do_attack_animation(src)
+
+		if(I_GRAB)
+			if (M == src)
+				return
+			if (!(status_flags & CANPUSH))
+				return
+			if(!incapacitated(INCAPACITATION_ALL) && (stance != STANCE_IDLE) && prob(grab_resist))
+				M.visible_message("<span class='warning'>[M] tries to grab [src] but fails!</span>")
+				return
+
+			var/obj/item/weapon/grab/G = new /obj/item/weapon/grab(M, src)
+
+			M.put_in_active_hand(G)
+
+			G.synch()
+			G.affecting = src
+			LAssailant = M
+
+			M.visible_message("<span class='warning'>[M] has grabbed [src] passively!</span>")
+			M.do_attack_animation(src)
+			ai_log("attack_hand() I was grabbed by: [M]",2)
+			pixel_x = old_x
+			react_to_attack(M)
+
+		if(I_HURT)
+			adjustBruteLoss(harm_intent_damage)
```

Comment:
I believe ..() could be called here instead of these five lines to cut down on code., as it should default back to https://github.com/VOREStation/VOREStation/blob/4faebdc4505fec02b61d08a267d549de981439a7/code/modules/mob/living/simple_animal/simple_animal.dm#L526\

Ex:
```c
		if(I_HURT)
			..()
			return
```